webpack.config.js

```diff
@@ -39,8 +39,7 @@ module.exports = {
         test: /\.tsx?$/,
         use: [
           !isProduction && {
-            loader: 'babel-loader',
-            options: { plugins: ['react-hot-loader/babel'] }
```

Comment:
why did you remove `react-hot-loader`?

Reply:
I intended to replace it with [React Fast Refresh](https://github.com/facebook/react/issues/16604), but it looks like it's still early days for that so I'll drop the commit and add React Hot Loading back